Makefile

```diff
@@ -97,17 +104,20 @@ DISTROS =           \
 	ubuntu-18.04    \
 
 # Template for running tests inside a Linux distro container
-DRUN = bin/ebash docker_run          \
-	--envlist "${ENVLIST}"           \
-	--nested                         \
-	--copy-to-volume "${PWD}:/ebash" \
-	--                               \
-	--init                           \
-	--tty                            \
-	--network host                   \
-	--privileged                     \
-	--rm                             \
-	--workdir /ebash                 \
+DRUN = bin/ebash docker_run                                 \
+	--envlist "${ENVLIST}"                                  \
+	--nested                                                \
+	--copy-to-volume   "ebash:${PWD}:/ebash"                \
+	--copy-from-volume "ebash:/ebash/.work:.work/docker/$1" \
+	--copy-from-volume-delete ".work/docker/$1/docker"      \
```

Comment:
The `--copy-from-volume` and `--copy-from-volume-delete` are new `docker_run` idioms. They allow you copy files out of the docker container before it is closed. And then to remove some part of the files that were copied back. This is because `docker cp` doesn't support an `--exclude` idiom

Reply:
Didn't know about those options. Good to know!